pkg/p2p/libp2p/libp2p.go

```diff
@@ -326,7 +326,7 @@ func (s *Service) Connect(ctx context.Context, addr ma.Multiaddr) (overlay swarm
 		return swarm.Address{}, err
 	}
 
-	if s.peers.addIfNotExists(stream.Conn(), i.Address) {
+	if exists := s.peers.addIfNotExists(stream.Conn(), i.Address); exists {
 		s.logger.Trace("skipped double connect")
```

Comment:
i still vote to get rid of this logline. it is not informative at all. it just says some general condition has happened. it neither mentions the base address of the reporting node, nor does it report with which node it skipped connecting to. in it's current form i would really not like to have such things in my logger output (as a developer)

Reply:
Deal!

Reply:
Is it ok to put it as debug? I would like to have it for testing?

Reply:
for testing you can add the line locally. in the old codebase we were at some points logging gigabytes per hour per cluster on `Trace`, these things must be taken into account. i prefer to log less but log meaningful things. either have this more informative as to what exactly happened or don't log at all

Reply:
ok

Reply:
also, i am not against logging. i am just arguing that it should be informative. so if you choose to log here you can do so as long as its informative and i'm totally cool with that. if you want to have a logline just add more info, that's all i'm asking